lib/util/accessibility-routing.js

```diff
@@ -2,17 +2,5 @@
  * Determine if an itinerary has accessibility scores
  */
 export const itineraryHasAccessibilityScores = (itinerary) => {
-  return !!itinerary.legs.find(leg => !!leg.accessibilityScore)
-}
-
-/**
- * Calculates the total itinerary score based on leg score by weighting
- * each leg equally
- */
-export const getAccessibilityScoreForItinerary = (itinerary) => {
-  const scores = itinerary.legs
-    .map((leg) => leg.accessibilityScore || null)
-    .filter((score) => score !== null)
-
-  return scores.reduce((prev, cur) => prev + (cur * (1 / scores.length)), 0)
+  return !!itinerary.legs.find((leg) => !!leg.accessibilityScore)
```

Comment:
Can the accessibility score of a leg be zero? if so, the whole itinerary score will NOT be displayed, and in such cases it should be displayed.

Reply:
Yes, understand what you mean now! Thank you, fixed in [92694f2](https://github.com/opentripplanner/otp-react-redux/pull/1092/commits/92694f2f351b69949b3ed95a9f9607c432f3fff5)